week-2/01-async-js/easy/readFromFile.js

```diff
@@ -0,0 +1,11 @@
+const fs = require("fs");
+
+fs.readFile("a.txt", "utf8", function (err, data) {
+let newData = ("   I hate noise and " + data+ "  ");;
+console.log(newData);
+  modifier(newData);
+});
+
```

Comment:
you are not doing expensive operation as mentioned in the question ....
you are simply reading the file